source/features/toggle-everything-with-alt.tsx

```diff
@@ -0,0 +1,83 @@
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+
+function getSimilarItems(item: HTMLElement): HTMLElement[] {
+	// Collapsed comments in PR conversations and files
+	if (item.matches('summary')) {
+		if ((item.parentElement as HTMLDetailsElement).open) {
+			return select.all('.minimized-comment details[open] summary');
+		}
+
+		return select.all('.minimized-comment details:not([open]) summary');
+	}
+
+	// "Load diff" buttons in PR files
+	if (item.matches('.js-diff-load')) {
+		return select.all('.js-file .js-diff-load');
+	}
+
+	// Review comments in PR
+	if (item.matches('.js-resolvable-thread-toggler')) {
+		const targets = select.all('.js-resolvable-thread-toggler');
+		if (select.exists('svg.octicon-unfold', item)) {
+			return targets.filter(target => !target.matches('.d-none') && select.exists('svg.octicon-unfold', target));
+		}
+
+		return targets.filter(target => !target.matches('.d-none') && select.exists('svg.octicon-fold', target));
+	}
+
+	// "Show comments" checkboxes
+	if (item instanceof HTMLLabelElement) {
+		const inputs = select.all<HTMLInputElement>('.js-file .dropdown-item .js-toggle-file-notes');
+		if ((item.control as HTMLInputElement).checked) {
+			return inputs.filter(input => !input.checked).map(input => input.labels![0]);
+		}
+
+		return inputs.filter(input => input.checked).map(input => input.labels![0]);
+	}
+
+	return [];
+}
+
+function handleEvent(event: DelegateEvent<MouseEvent, HTMLElement>): void {
+	if (!event.altKey) {
+		return;
+	}
+
+	const clickedItem = event.delegateTarget;
+	const viewportOffset = clickedItem.parentElement!.getBoundingClientRect().top;
+	const similarItems = getSimilarItems(clickedItem);
+
+	for (const item of similarItems) {
+		if (item !== clickedItem) {
+			item.click();
+		}
+	}
+
+	// Scroll to original position where the click occurred after the rendering of all click events is done
+	requestAnimationFrame(() => {
+		const newOffset = clickedItem.parentElement!.getBoundingClientRect().top;
+		window.scrollBy(0, newOffset - viewportOffset);
+	});
+}
+
+function init(): void {
+	delegate('.repository-content', '.minimized-comment details summary', 'click', handleEvent);
+	delegate('.repository-content', '.js-file .js-diff-load', 'click', handleEvent);
+	delegate('.repository-content', '.js-file .dropdown-menu label.dropdown-item:first-child', 'click', handleEvent);
+	delegate('.repository-content', '.js-file .js-resolvable-thread-toggler', 'click', handleEvent);
+}
+
+features.add({
+	id: 'toggle-everything-with-alt',
+	load: features.onAjaxedPages,
+	description: 'Toggle all similar items while holding `alt`',
```

Comment:
I think we can list all supported "items" both here and in the readme, e.g. `Toggle all items when alt-clicking minimized commments, deferred diffs, hidden comments in the PR Files tab"

Reply:
How about
> Toggle all similar items (minimized comments, deferred diffs, etc) while holding `alt`